### ISSUE TITLE
hypercoagulability and coagulopathy

### ISSUE THREAD

**Issue by anna-anagnostop** [obophenotype/mammalian-phenotype-ontology#3329]
Need a new term for hypercoagulability (a child of abnormal blood coagulation MP:0002551) 

hypercoagulability
DEF: a state or condition marked by an increased tendency to form blood clots within blood vessel(s)
SYN: thrombophilia
SYN: hypercoagulability state
SYN: excessive blood clotting
SYN: excessive blood coagulability
SYN: prothrombotic state  -- all of these are exact
SOURCE: https://www.merriam-webster.com/dictionary/hypercoagulability, [HP:0100724](http://www.ontobee.org/ontology/HP?iri=http://purl.obolibrary.org/obo/HP_0100724)

NOTE: HP:0100724 uses Blood hyperviscosity as an exact synonym. MP has a term for abnormal blood viscosity [MP:0000234](http://www.informatics.jax.org/vocab/mp_ontology/MP:0001598); hence, blood hyperviscosity will NOT be used as a synonym for hypercoagulability.

Regarding coagulopathy (Taber's def: a defect in blood-clotting, esp. one that results in abnormal bleeding or thrombus formation. Laboratory evidence of coagulopathy includes prolongation of the prothrombin time, international normalized ratio, partial thromboplastin time, or a decrease in the number of circulating platelets).  

Make coagulopathy a _related_ synonym of abnormal blood coagulation MP:0002551 as certain types of coagulopathy do not allow blood to clot while others are characterized by hypercoagulation (see https://www.topdoctors.co.uk/medical-dictionary/autoimmune-coagulation-disorders and DOID:0111908)

Also, check annotations under MP:0020215

**Comment by anna-anagnostop** [comment]
Added new term:
hypercoagulability MP:0031214

Make coagulopathy a related synonym of abnormal blood coagulation MP:0002551

**Comment by anna-anagnostop** [comment]
Reopened issue as textual definition for MP:0031214 does not show up on public MP Browser. Definition was erroneously entered under "Description" in Protege; this needs to be fixed
